data/changelog/posts/ocaml-lsp/2025-08-20-ocaml-eglot-brings-lsp-support-to-emacs.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Emacs Integration for OCaml LSP Server: Introducing ocaml-eglot
+tags: [ocaml-lsp, platform]
+---
+
+# Emacs Integration for OCaml LSP Server: Introducing ocaml-eglot
+
+## TL;DR
+
+`ocaml-eglot` provides full OCaml language support in Emacs through the Language Server Protocol (LSP) instead of direct Merlin integration. It offers the same features as `merlin.el` with simplified setup and enhanced capabilities like project-wide search. If you're starting fresh or want a more standardized approach, try `ocaml-eglot`. If your current `merlin.el` setup works well, you can continue using it.
+
+**Quick start**: Install `ocaml-lsp-server`, add `ocaml-eglot` to your Emacs config, get the same OCaml development experience with less configuration.
+
+## What is ocaml-eglot?
+
+`ocaml-eglot` connects Emacs to `ocaml-lsp-server` using the Language Server Protocol, providing a standardized way to get OCaml language support. This eliminates editor-specific integrations while preserving all OCaml-specific features.
+
+## Who Should Use ocaml-eglot?
+
+**Use ocaml-eglot if you:**
+- Are starting fresh with OCaml in Emacs
+- Want simplified configuration with automatic setup
+- Use multiple editors and want consistent OCaml support
+- Want access to project-wide search and rename features
+
+**Stick with merlin.el if:**
+- Your current setup is working perfectly and heavily customized
+- You prefer direct Merlin communication
+
+Both approaches are fully supported.
```

Comment:
I'm not really sure what ‘fully supported’ means in this context. I think we can say that, for the moment, we don't plan to provide any special support for merlin.el unless we receive a lot of requests.